core/src/main/scala/cats/data/XorT.scala

```diff
@@ -65,6 +65,9 @@ final case class XorT[F[_], A, B](value: F[A Xor B]) {
 
   def bimap[C, D](fa: A => C, fb: B => D)(implicit F: Functor[F]): XorT[F, C, D] = XorT(F.map(value)(_.bimap(fa, fb)))
 
+  def bitraverse[G[_], C, D](f: A => G[C], g: B => G[D])(implicit traverseF: Traverse[F], bitraverseXor: Bitraverse[Xor], applicativeG: Applicative[G]): G[XorT[F, C, D]] =
```

Comment:
Yeah, that felt strange. I wrote it this way because of the `traverseXorA` parameter in [`XorT.traverse`](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/XorT.scala#L99).
Can I just write `Bitraverse[Xor].bitraverse(axb)(f, g)` directly ?
Can I do the same with [`Bifoldable[Xor]`](https://github.com/typelevel/cats/pull/986/files#diff-443b0c0bbff3ee4ef3a4b8805699f07cR343) in `XorTBifoldable` ?